Bugsnag/BugsnagSessionTracker.m

```diff
@@ -117,32 +115,22 @@ - (void)setCodeBundleId:(NSString *)codeBundleId {
 
 #pragma mark - Creating and sending a new session
 
-- (void)startNewSession {
-    [self startNewSessionWithAutoCaptureValue:NO];
-}
-
 - (void)pauseSession {
-    [[self currentSession] stop];
+    self.currentSession.stopped = YES;
```

Comment:
It feels weird to reach into a session from another object to change its internal state rather than encapsulating start/stop/resume in BugsnagSession proper.

Reply:
I can see the argument, but it's not unusual to set the value of an object's property. The `stopped` property is also not really "internal" since it's declared in the header. Furthermore the session object never examines it, only `BugsnagSessionTracker` does.

Reply:
It's not the end of the world, but the fact that a property defining the session's state is writable, and is only set externally by an administrator object, is a code smell. Normally an object would manage its own state internally and present a view to the world for inspecting (not directly setting) that state.